pkg/api/conntrack.go

```diff
@@ -83,6 +82,12 @@ type ConnTrackOperationEnum struct {
 	Max   string `yaml:"max" doc:"max"`
 }
 
+type ConnTrackSchedulingGroup struct {
+	Selector                 map[string]string `yaml:"selector,omitempty" doc:"key-value map to match against connection fields to apply this scheduling"`
```

Comment:
Selector matches only string: string. Is this the intended usage? So we cannot, for instance, select on integer values?

Reply:
We can select integer values. The integer values are [converted](https://github.com/netobserv/flowlogs-pipeline/pull/357/files#diff-bd390b4fc85fff1d1e1532f7a0077a03f1e34265d04cbe4ace449fb22e6a2d6fR113) to string before the comparison.

At first, I used `map[string]interface{}`. But then I noticed that setting the selector to `Proto: 6` is parsed as `int`. The problem is that the grpc ingester [parses](https://github.com/netobserv/flowlogs-pipeline/blob/30d42fd09a981391ee6ea52ef6dc18786474509c/vendor/github.com/netobserv/netobserv-ebpf-agent/pkg/pbflow/flow.pb.go#L547) `Proto` as `uint32`. Comparing `int` and `uint32` is evaluated to false regardless of the values. That's why I convert and compare strings.